src/unit3/lesson2/README.md

```diff
@@ -23,7 +23,7 @@ Start Visual Studio and create a new `Console Application Project` named
 command:
 
 ```powershell
-Install-Package RavenDB.Client -Version 4.0.3
+Install-Package RavenDB.Client -Version 5.1.8
```

Comment:
Can you add `MinimumVersion` and `MaximumVersion` option here?

```suggestion
Install-Package RavenDB.Client -MinimumVersion 5.1.0 -MaximumVersion 5.9.9
```

And in the other place as well please.

Reply:
The `MinimumVersion` and `MaximumVersion` are not supported, as well as wildcards.
For .NET Core based projects `dotnet add package --version 5.*` can be used.

See: https://stackoverflow.com/questions/49795616/nuget-install-package-with-wildcard-version

Reply:
It's there in the docs - should be supported. What's your PS version?

Reply:
Ah, it says there on top it needs to be run within Package Manager Console in Visual Studio. Hmm

Reply:
Ok it doesn't work for me either.. Let's make it `5.2.0` and move along.

Reply:
Done